wastewater/dataModel.WasteWaterTreatment/WasteWaterTreatment-schema.json

```diff
@@ -0,0 +1,760 @@
+                    "type": "number"
+                },
+                "unitCode": {
+                    "type": "string"
+                },
+                "observedAt": {
+                    "type": "string",
+                    "format": "date-time"
+                }
+            },
+            "required": [
+                "type",
+                "value"
+            ]
+        },
+        "pressure": {
+            "type": "object",
+            "description": "Property. Pressure measurement in the blower",
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "enum": [
+                        "Property"
+                    ]
+                },
+                "value": {
+                    "type": "number"
+                },
+                "unitCode": {
+                    "type": "string"
+                },
+                "observedAt": {
+                    "type": "string",
+                    "format": "date-time"
+                }
+            },
+            "required": [
+                "type",
+                "value"
+            ]
+        },
+        "airflowEstimation": {
+            "type": "object",
+            "description": "Property. Airflow estimation by AI soft sensor simulation.",
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "enum": [
+                        "Property"
+                    ]
+                },
+                "value": {
+                    "type": "number"
+                },
+                "unitCode": {
+                    "type": "string"
+                },
+                "observedAt": {
+                    "type": "string",
+                    "format": "date-time"
+                }
+            },
+            "required": [
+                "type",
+                "value"
+            ]
+        },
+        "status": {
+            "type": "object",
+            "description": "Property. Status of a Valve.",
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "enum": [
+                        "Property"
+                    ]
+                },
+                "value": {
+                    "type": [
+                        "number",
+                        "string",
+                        "array"
+                    ],
+                    "enum": [
+                        "OPEN",
+                        "CLOSED",
+                        "CV"
+                    ]
+                },
+                "unitCode": {
+                    "type": "string"
+                },
+                "observedAt": {
+                    "type": "string",
+                    "format": "date-time"
+                }
+            },
+            "required": [
+                "type",
+                "value"
+            ]
+        },
+        "tag": {

```

Comment:
Subproperties defined in the NGSI LD standard are not necessary (i.e. unitcode, observedAt, etc) it should be removed